src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.epics.js

```diff
@@ -2,10 +2,14 @@
 import { ofType } from 'redux-observable';
 import uuid from 'uuid/v4';
 import { map } from 'rxjs/operators';
-import { scheduleEventWidgetActionTypes, scheduleEvent } from './WidgetEventSchedule.actions';
+import {
+    scheduleEventWidgetActionTypes,
+    scheduleEvent,
+    updateScheduledDateForEvent,
+} from './WidgetEventSchedule.actions';
+import { statusTypes } from '../../events/statusTypes';
 
-
-export const scheduleNewEnrollmentEventEpic = (action$: InputObservable) =>
+export const scheduleNewEnrollmentEventEpic = (action$: InputObservable, store: ReduxStore) =>
```

Comment:
NIT: Both schedule and re-schedule will use this function so maybe `scheduleEnrollmentEventEpic` is a more suitable name for this epic.